packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -209,7 +215,7 @@ export class FieldSchemaAlpha<Kind extends FieldKind = FieldKind, Types extends
     readonly allowedTypesMetadata: AllowedTypesMetadata;
     // (undocumented)
     readonly annotatedAllowedTypes: ImplicitAnnotatedAllowedTypes;
-    get annotatedAllowedTypeSet(): ReadonlyMap<TreeNodeSchema, AllowedTypeMetadata>;
+    get annotatedAllowedTypeSet(): readonly AnnotatedAllowedSchema[];
```

Comment:
The naming if these isn't great. `annotatedAllowedTypeSet` is not a set.

Maybe `annotatedAllowedTypesNormalized` would make more sense.

Or maybe `annotatedAllowedTypesInput: ImplicitAnnotatedAllowedTypes` and `annotatedAllowedTypes: readonly AnnotatedAllowedSchema[]`.

Might be worth going over it with one of our API reviewers (Noah or Josh) and see if they have ideas.

Regardless, you did actually change the alpha API surface here, so a simple changeset noting that should be included.